### DISCUSSION TITLE
How do I implement for all repos in my org

### DISCUSSION THREAD

**Discussion by Chris-Sheridan** [actions/runner#3153] [General]
I created an image based on this and was able to successfully run a job that deployed to my Kubernetes cluster for an individual repo.  However, I want configure this for my org so one implementation will handle all repos?  Is there documentation for this?

What environment variables/args need to be set for a self-hosted running to be able to work with all repos in an org?  For example, I assume RUNNER_SCOPE should = "org" and I should have my org set for ORG_NAME.  What else do I need to do?  Thanks.

**Comment by crohr** [comment]
@Chris-Sheridan you might be interested in [RunsOn](https://runs-on.com). The RunsOn service can handle runners for any number of repositories within your organisation.
